docs/UserGuide.md

```diff
@@ -44,179 +44,246 @@ faster than traditional GUI apps.
 
-Example: `done 2`
-* Mark the second order in the list of orders as complete.
+Marks an incomplete order as complete from CHIM, identified by its index.
 
-### Finding customers: `findcustomer`
+This assigns cheeses to the order.
 
-Searches for a customer in CHIM.
+Format: `done ORDER_INDEX`
+* The `ORDER_INDEX` provided must be a positive integer.
 
-Format: `findcustomer [n/NAME_KEYWORDS...] [p/PHONE_KEYWORDS...] [e/EMAIL_KEYWORDS...] [a/ADDRESS_KEYWORDS...] `
-* Search is case-insensitive, e.g. Betty will match betty.
-* Search will search by given keywords as prefix, e.g. Bet will match Betty.
+Example: `done 2`
+* Marks the second order in the list of orders as complete.
 
-Examples:
-* `findcustomer n/Betty`
-* `findcustomer a/Clementi Road`
+#### Deleting an order: `deleteorder`
 
-### Finding cheeses: `findcheese`
+Deletes a specific order from CHIM, identified by its index.
 
-Searches for particular cheeses in CHIM.
+If the order is marked as complete and has cheeses assigned to it, those cheeses will be deleted by CHIM.
 
-Format: `findcheese [t/CHEESE_TYPE_KEYWORDS...] [s/ASSIGNMENT_STATUS]`
-* Search is case-insensitive, e.g. Brie will match brie.
-* Assignment status parameter must be either 'assigned' or 'unassigned'.
+Format: `deleteorder ORDER_INDEX`
+* The `ORDER_INDEX` provided must be a positive integer.
 
-Examples:
-* `findcheese t/Brie Feta`
-* `findcheese s/unassigned`
-* `findcheese t/Gouda s/assigned`
+Example: `deleteorder 2`
 
-### Finding orders: `findorder`
+#### Finding orders: `findorder`
 
 Searches for particular orders in CHIM.
 
-Format: `findorder [t/CHEESE_TYPE_KEYWORDS...] [n/CUSTOMER_NAME_KEYWORDS] [p/CUSTOMER_PHONE_KEYWORDS] [s/COMPLETION_STATUS]`
+Format: `findorder [t/CHEESE_TYPE_KEYWORDS…] [n/CUSTOMER_NAME_KEYWORDS…] [p/CUSTOMER_PHONE_KEYWORDS…] [s/COMPLETION_STATUS]`
+* At least one of the optional fields must be provided to find an order.
 * Search is case-insensitive, e.g. Brie will match brie.
-* Assignment status parameter must be either 'complete' or 'incomplete'.
+* Assignment status parameter must be either `complete` or `incomplete`.
+* Search will find an order which matches **all** of the given fields,
+e.g. `findorder t/Brie s/incomplete` will find an order with cheese type Brie and is incomplete.
 
 Examples:
 * `findorder t/Brie Feta s/incomplete`
 * `findorder n/Alice s/incomplete`
 * `findorder p/92280919`
 
+
 ### Clearing all data in CHIM: `clear`
 
-Clears all data in CHIM and save files 
+Clears all data in CHIM and save files.
 
 Format: `clear`
 
+
 ### Exiting the program: `exit`
 
 Exits the program.
 
 Format: `exit`
 
 
+### Saving the data
+
+CHIM data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+
+
+### Editing the data file
+
+CHIM data are saved as a JSON file `[JAR file location]/data/chim.json`. Advanced users are welcome to update data directly by editing that data file.
```

Comment:
May I confirm if this is the case for our project? I tried running the JAR file but couldn't find the data folder anywhere. The storage works perfectly though.

Reply:
The data folder appears for me.

Reply:
its a hidden folder .data/chim,json

Reply:
okay got it thanks